activities/PediatricScreener-Parent/pediatric_screener_parent_schema.jsonld

```diff
@@ -0,0 +1,192 @@
+  "skos:altLabel": "pediatric_screener_parent_schema",
+  "schema:description": "Respond to questions about your child as part of this mental health screener.",
+  "schema:schemaVersion": "0.0.1",
+  "schema:version": "0.0.1",
+  "preamble": "",
+  "branchLogic": {},
+  "scoringLogic": {},
+  "repronim:timeUnit": "yearmonthdate",
+  "variableMap": [
+    {
+      "variableName": "instructions_1",
+      "isAbout": "instructions_1"
+    },
+    {
+      "variableName": "sad_or_unhappy",
+      "isAbout": "sad_or_unhappy"
+    },
+    {
+      "variableName": "hopeless",
+      "isAbout": "hopeless"
+    },
+    {
+      "variableName": "down_on_self",
+      "isAbout": "down_on_self"
+    },
+    {
+      "variableName": "worry",
+      "isAbout": "worry"
+    },
+    {
+      "variableName": "having_less_fun",
+      "isAbout": "having_less_fun"
+    },
+    {
+      "variableName": "fidgety",
+      "isAbout": "fidgety"
+    },
+    {
+      "variableName": "daydream_too_much",
+      "isAbout": "daydream_too_much"
+    },
+    {
+      "variableName": "distracted_easily",
+      "isAbout": "distracted_easily"
+    },
+    {
+      "variableName": "trouble_concentrating",
+      "isAbout": "trouble_concentrating"
+    },
+    {
+      "variableName": "driven_by_motor",
+      "isAbout": "driven_by_motor"
+    },
+    {
+      "variableName": "fight_other_children",
+      "isAbout": "fight_other_children"
+    },
+    {
+      "variableName": "not_listen_to_rules",
+      "isAbout": "not_listen_to_rules"
+    },
+    {
+      "variableName": "not_understand_others_feelings",
+      "isAbout": "not_understand_others_feelings"
+    },
+    {
+      "variableName": "tease_others",
+      "isAbout": "tease_others"
+    },
+    {
+      "variableName": "blame_others_for_troubles",
+      "isAbout": "blame_others_for_troubles"
+    },
+    {
+      "variableName": "refuse_to_share",
+      "isAbout": "refuse_to_share"
+    },
+    {
+      "variableName": "take_things",
+      "isAbout": "take_things"
+    },
+    {
+      "variableName": "bullied",
+      "isAbout": "bullied"
+    },
+    {
+      "variableName": "better_off_dead",
+      "isAbout": "better_off_dead"
+    },
+    {
+      "variableName": "little_interest_or_pleasure",
+      "isAbout": "little_interest_or_pleasure"
+    },
+    {
+      "variableName": "depressed_or_hopeless",
+      "isAbout": "depressed_or_hopeless"
+    },
+    {
+      "variableName": "instructions_2",
+      "isAbout": "instructions_2"
+    },
+    {
+      "variableName": "worry_about_weight_or_getting_fat",
+      "isAbout": "worry_about_weight_or_getting_fat"
+    },
+    {
+      "variableName": "throw_up",
+      "isAbout": "throw_up"
+    },
+    {
+      "variableName": "control_weight",
+      "isAbout": "control_weight"
+    },
+    {
+      "variableName": "eating_binges",
+      "isAbout": "eating_binges"
+    }
+  ],
+  "ui": {
+    "order": [
+      "instructions_1",
```

Comment:
@sanuann -- So the preamble automatically becomes an information screen that precedes a sequence of items?  This is fine if there is only one set of instructions for all of the questions, but here we have multiple instructions, each one to precede a subset of questions.  How would the preamble take this into account?

Reply:
The subset of questions having its own instruction should be defined as an `activity` with the instruction as it's preamble.

Reply:
The idea is: an activity can contain a set of items and/or other sub-activities.

Reply:
AH!  I see.  I only have experience building out the pediatric screeners applet at the jsonld level.  How do I nest activities?  Is this something @akeshavan's python script can handle?

Reply:
U list the order of items and sub-activities in ui.order of the main activity. Check out https://github.com/ReproNim/schema-standardization/blob/9bcde5ef6c2694febc3e741e8610c406ae53cefd/activities/VoiceScreening/voice_screening_schema.jsonld#L25

Reply:
I am not sure about her python script. Can check in some time and let you know.

Reply:
Thank you, @sanuann!

Reply:
@binarybottle at the moment, the python script doesn't allow adding sub-activities. 
it shouldn't be very difficult to make the above changes manually though. you just need to create activity schemas for the sub-activities and refer to them in ui.order of the main activity.

Reply:
@sanuann -- @akeshavan informs me that nested activities is only relevant for the web version of MindLogger.  Given this, do I need to do anything for this PR?

Reply:
i still think, instructions shouldn't be defined as an `item`. @satra can you please look into this and correct me if I am wrong?

Reply:
instructions should indeed go into preamble

Reply:
@satra -- If I have three sets of instructions for three subsets of questions, what goes in the preamble?

Reply:
then each subset of questions is an activity in the schema.

Reply:
also if these questions are really coming from other questionnaires, then they should be first coded as those questionnaires. i'll take a look at lindsay's emails more closely tonight.

Reply:
@binarybottle see the `markdown-message` example item here: https://github.com/ReproNim/schema-standardization/blob/master/activities/MindLoggerDemo/items/markdown-message.jsonld

@satra @sanuann mindlogger-app doesn't have support for sub-activities yet. Since we don't have screen real-estate for the preamble/instruction, Stu created a `markdown-message` inputType, and this is something the app can render.

Reply:
@binarybottle I think this whole "pediatric screener" needs to be refactored. It only makes sense to have parent- and self-report in the same activity set if we want an informant to talk about themselves and their child, and I don't think we're talking about kids with kids or kids who are their own kids. The parent-report should be one activity set with all of the parent-report activities, and the self-report should be a different activity set with all the self-report activities.

Also, the preamble is a feature we still need to build for the app; I know you said, "Let's just make sure that we don't forestall a partner's use of MindLogger based on a minor missing functionality," but I think we don't have a value-add if our schema is nonsensical or inconsistent, ie, we could work around this deficit with a `markdown-message`, but I think we lose MindLogger's differentiating value when we do things like that. I vote we take the time to encode semantically and add the interface features as we need them.

Reply:
@shnizzedy - one possibility is to insert a question at the beginning such as  "Are you a parent?" and then drive the set of questions following it.

that being said lindsay was going to walk us through the workflow.

Reply:
Thank you @satra and @sanuann for clarifying what we want (nested or sequences of activities, each with its own preamble, or sourcing questions from their original questionnaires), but given what @akeshavan said about mindboggle-app's current limitations, how should we proceed?  I would like to build other questionnaires with multiple components like the ksads and winston prep.

Reply:
@binarybottle - i think there are two things to consider. the schema is somewhat independent of mindlogger. the intent is to encode things in a way that makes linked sense and that people can reuse. this may highlight current limitations of mindlogger which should be resolved or hacked.

in the present scenario this little survey presents some unique challenges. the questions come from some underlying questionnaire, but they have been modified. for example the middle three questions come from PHQ9, but the preamble/instructions refers to a 6 month time span. there is also the fact that there is a child and a parent questionnaire and that either questionnaire can be given in english or spanish. the answers to these questionnaires could trigger other questionnaires. 

this is a good test for schema, mindlogger etc.,.

i would like to see the activity set and the activities represent the best way we can represent these questions (and link to their original variants or originals). then this will highlight what the difference is between the web version and the app version. one can then tackle those.

@binarybottle - would it be possible to cherrypick your pediatric screener comments into a separate branch and send a PR from that branch? this would also help us focus on the screener as opposed to all the nice activity sets you have created. we would be happy to review those as well, but perhaps we can focus on the pediatric screener.